.travis.yml

```diff
@@ -85,7 +85,7 @@ matrix:
       env:
         - "GEM=ar:mysql2 MYSQL=mariadb"
       addons:
-        mariadb: 10.0
+        mariadb: 10.2
```

Comment:
This change has already exist in master.

Reply:
Thanks. rebased from the current master.